### PR TITLE
Add CI test verify Quay AutoSD QM is working

### LINKED ISSUE #236: Add CI test verify Quay AutoSD QM is working
Add tier-1 tmt test 
Verifies QM https://quay.io/repository/centos-sig-automotive/autosd
is working

### PR DESCRIPTION
**Pull request by Yarboa**
Resolves #236

Add tier-1 tmt test
Verifies QM https://quay.io/repository/centos-sig-automotive/autosd
is up and running

### CODE REVIEW

**Comment by Yarboa** [comment]
@dougsland need to add to packit

File: tests/e2e/tier-1.fmf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+   # Can not use provision no podman args
+   how: local
+
+adjust:
+   prepare+:
+      - name: Prepare Repos
+        how: shell
+        script: |
+           podman run --replace -d --name autosd --security-opt label=disable \
+                  --cap-add=sys_admin,mknod -it \
```
Comment (Yarboa) [inline tests/e2e/tier-1.fmf:17]:
```suggestion
                  --cap-add=sys_admin -it \
```

Comment (dougsland) [reply tests/e2e/tier-1.fmf:17]: Sure, let me ask the podman master, @rhatdan WDYT

Comment (Yarboa) [reply tests/e2e/tier-1.fmf:17]: Currently there is another issue, I will stick to --privileged inside for now

Comment (dougsland) [reply tests/e2e/tier-1.fmf:17]: sure thing, could you please move this one to `Draft` until things get in the shape?

**Comment by Yarboa** [comment]
@dougsland PTAL

**Pull request merged**